src/includes/docs-nav.html

```diff
@@ -46,6 +46,7 @@
 		<li><a href="/docs/logging">How Logging Works</a></li>
 		<li><a href="/docs/metrics">Monitoring Caddy</a></li>
 		<li><a href="/docs/architecture">Caddy Architecture</a></li>
+		<li><a href="/docs/service">Linux Service</a></li>
```

Comment:
```suggestion
		<li><a href="/docs/service">System service</a></li>
```

And I wonder if this should be in the "Get Caddy" section, but I can see why it's good in "Articles" too as it's not really about "getting" Caddy...

Reply:
All the other "articles" use Title Case, should we not do the same for this page? i.e. `System Service`

Reply:
Yes, I suppose we should. :+1: